pr_testing/test-cmssw-provides.sh

```diff
@@ -36,6 +36,10 @@ fi
 sed -i -e "s!@release@!${WORKSPACE}/${CMSSW_RELEASE}!g" $CMS_BOT_DIR/pr_testing/cmssw-pr-package.spec
 cp $CMS_BOT_DIR/pr_testing/cmssw-pr-package.spec cmsdist/
 
+pkgtools/cmsBuild --repo cms.week${WEEK_NUM} -a $SCRAM_ARCH -c cmsdist -i build --builders 1 -j 8 build rpm
+RPM_DIR=$(ls -1tr build/${SCRAM_ARCH}/external/rpm/|tail -1)
+source build/${SCRAM_ARCH}/external/rpm/$RPM_DIR/etc/profile.d/init.sh
```

Comment:
why installing `rpm` here? 
You are isntalling `cms-common` later so that will install `rpm and cmspkg`. After `build cms-common` command you can use `./build/cmspkg -a $SCRAM_ARCH env -- rpm <options>` to run any rpm command :-)

Reply:
you can run `/cvmfs/cms-ib.cern.ch/sw/$(uname -m)/week${WEEK_NUM}/common/cmspkg -a $SCRAM_ARCH env -- rpm -q --provides cms+cmssw+<version>`  to get the provides

Reply:
I think you need to get provides for 
- `cms+cmssw+${CMSSW_RELEASE}` if dev area is based on full release
- `cms+cmssw-patch+${CMSSW_RELEASE}` and `cms+cmssw+CMSSW_<full release>` if dev area is based on a patch relesse

Reply:
but all this is needed if you have use local RPM DB from the PR build. If  RPM DB was copied for cvmfs then it should already contian all the provides for cmssw and cmssw-patch

Reply:
I was sure I'd doing something weird, but couldn't figure out the right way. Thanks!